src/poonamDPracticeTest/StringNumLetters.java

```diff
@@ -0,0 +1,65 @@
+package poonamDPracticeTest;
+/*
+ * Write a program to print - 
+   1. Capital letters; 2. odd number sum; 3. Special characters; 4. Even number sum; 6. small letters
+   output - ABF4.[][%&2hs
+ */
+public class StringNumLetters {
+	static void printLettersNum(String str)
+	{
+		String Num = "";
+		String SmallLetters = "";
+		String CapitalLetters = "";
+		String SpclChar = "";
+		int OddSum = 0 ;
+		int EvenSum = 0;
+		for(int i=0;i<str.length();i++)
+		{ 
+			char ch = str.charAt(i);
+			boolean isNum = true;
+			boolean isLetter = true;
+			if(isNum = Character.isDigit(ch))
```

Comment:
Can't we write directly if(Character.isDigit(ch)), why do we need to take it in boolean

Reply:
I was trying to print all the letters & numbers at first; 
I'll update the program again